src/sentry/static/sentry/app/views/alerts/details/metricChart.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import LineChart from 'app/components/charts/lineChart';
+import space from 'app/styles/space';
+import {Series} from 'app/types/echarts';
+
+type Props = {
+  data: Series[];
+};
+
+const MetricChart = ({data}: Props) => {
+  return (
+    <LineChart
```

Comment:
eventually we will change this component to also handle highlighting of green, yellow, red areas based on the metric triggers